qemu/tests/blockdev_mirror_qemuio_target.py

```diff
@@ -0,0 +1,51 @@
+from avocado.utils import process
+
+from virttest import utils_misc
+
+from provider.blockdev_mirror_nowait import BlockdevMirrorNowaitTest
+
+
+class BlockdevMirrorQemuioTarget(BlockdevMirrorNowaitTest):
+    """
+    Qemuio target image during mirror
+    """
+
+    def qemuio_target_image(self):
+        tag = self._target_images[0]
+        image_params = self.params.object_params(tag)
+        image = self.disk_define_by_params(image_params, tag)
+        filename = image.image_filename
+        fmt = image.image_format
+        qemu_io = utils_misc.get_qemu_io_binary(self.params)
+        qemuio_cmd = self.params.get("qemuio_cmd") % (qemu_io, fmt, filename)
+        try:
+            process.run(qemuio_cmd, shell=True)
+        except process.CmdError as e:
+            if self.params["error_msg"] not in e.result.stderr.decode():
+                self.test.fail("Write to image that using by another process")
```

Comment:
Suggest to print the e.result.stderr here if we happen to hit an unknown error or if the err msg changes

Reply:
Have updated, thanks @zhencliu